### PR TITLE
60 admin user dashboard

### PR DESCRIPTION
**Pull request by meyerhoferc**
* adds role to users (default + admin)
* adds admin::base & admin::orders controllers
* adds routes for admin dashboard
* user only sees admin dashboard if logged in & has role of admin

all tests pass

### CODE REVIEW

File: spec/features/users/admin_sees_dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  it "as a registered user" do
+    user = User.create(first_name: "Charlotte",
+                       last_name: "Moore",
+                       username: "Cj",
+                       email: "email@email.com",
+                       password: "password")
+
+    expect(user.admin?).to eq(false)
```
Comment (CjMoore) [inline spec/features/users/admin_sees_dashboard_spec.rb:29]: 👍

File: spec/features/users/admin_sees_dashboard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    end
+  end
+
+  it "as an unregisterd user" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(nil)
+
+    visit '/admin/dashboard'
+
+    within("h1") do
+      expect(page).to have_content("Error: 404 page not found")
```
Comment (CjMoore) [inline spec/features/users/admin_sees_dashboard_spec.rb:45]: 👍

**Review by CjMoore — APPROVED** [review]
Looks good to me. Ready for second review and merge @valerietrudell @cmacaulay

**Comment by valerietrudell** [comment]
Looks good!

**Pull request merged**